larrecodnn/CVN/interfaces/ICVNMapper.cxx

```diff
@@ -0,0 +1,59 @@
+#include "larrecodnn/CVN/interfaces/ICVNMapper.h"
+
+namespace lcvn {
+
+  template <class T, class U>
+  ICVNMapper<T, U>::ICVNMapper(fhicl::ParameterSet const& pset)
+    : EDProducer{pset}
+    , fHitsModuleLabel(pset.get<std::string>("HitsModuleLabel"))
+    , fClusterPMLabel(pset.get<std::string>("ClusterPMLabel"))
+    , fMinClusterHits(pset.get<unsigned short>("MinClusterHits"))
+    , fProducer(pset.get<fhicl::ParameterSet>("PixelMapProducer"))
+  {
+
+    produces<std::vector<lcvn::PixelMap>>(fClusterPMLabel);
+  }
+
+  //......................................................................
+  template <class T, class U>
+  ICVNMapper<T, U>::~ICVNMapper()
+  {
+    //======================================================================
+    // Clean up any memory allocated by your module
+    //======================================================================
+  }
+
+  //......................................................................
+  template <class T, class U>
+  void ICVNMapper<T, U>::beginJob()
+  {}
+
+  //......................................................................
+  template <class T, class U>
+  void ICVNMapper<T, U>::endJob()
+  {}
```

Comment:
See other comments about removing empty destructor and `beginJob`/`endJob` functions.